### ISSUE TITLE
Legend of symbols

### ISSUE THREAD

**Issue by myadhdoutlet** [veltzer/openbook#96]
This is a great project; much thanks for all of the great work. Might you consider adding a symbol key in a future revision? It would help us newer jazz cats.

**Comment by veltzer** [comment]
What do you mean a symbol key? You mean an explanation of the various chord types?

**Comment by myadhdoutlet** [comment]
Yes, that's what I mean... and yes, the size and quality of PDF is great! 

I was able to look everything up elsewhere once I started going through the book, but it would be nice to point out that for instance, the triangle represents a Major 7th chord. That's the only one I recall confusing me, but there may be other notations I haven't seen yet. 

Anyhow, just a friendly suggestion. Great job!

**Comment by veltzer** [comment]
I'm summarizing: Add a page or two describing all the chord types.
